package.json

```diff
@@ -76,7 +76,7 @@
     "typescript": "3.0.3"
   },
   "dependencies": {
-    "@elastic/charts": "^18.2.2",
+    "@elastic/charts": "^19.2.0",
```

Comment:
can you try removing this dependency? After removing it, we should automatically use [elastic chart of Kibana](https://github.com/elastic/kibana/blob/master/package.json#L124) instead

Reply:
Good point. Sorry that I missed this reply. Will remove it in next PR.